dist/betteR20-5etools.user.js

```diff
@@ -2786,7 +2788,31 @@ function baseTool () {
 						else return getAllJournalItems().filter(it => it);
 					}
 
+					// Allow for deleting tables as well
+					function getRollableTables() {
+						const tItems = [];
+						if ($cbTables.prop("checked")) {
```

Comment:
Minor styling thing: flip this to be
```js
{
  if (!$cbTables.prop("checked")) return [];
  
  // ... rest of function body ... 
}
```
Means one less level of nesting, and one less thing to mentally track when reading the function. "Return early"!